mmcls/models/backbones/levit.py

```diff
@@ -0,0 +1,586 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import itertools
+
+import torch
+import torch.nn as nn
+from mmcv.cnn import Linear, build_conv_layer, build_norm_layer
+from mmengine.model import BaseModule
+
+from mmcls.models.backbones.base_backbone import BaseBackbone
+from mmcls.models.heads.levit_head import LeViTClsHead
+from mmcls.registry import MODELS
+
+
+class hybrid_cnn(BaseModule):
+
+    def __init__(
+            self,
+            embed_dim,
+            kernel_size=3,
+            stride=2,
+            pad=1,
+            dilation=1,
+            groups=1,
+            bn_weight_init=1,
+            activation=nn.Hardswish,
+            conv_cfg=None,
+            norm_cfg=dict(type='BN'),
+            init_cfg=None,
+    ):
+        super(hybrid_cnn, self).__init__(init_cfg=init_cfg)
+
+        self.input_channels = [
+            3, embed_dim // 8, embed_dim // 4, embed_dim // 2
+        ]
+        self.output_channels = [
+            embed_dim // 8, embed_dim // 4, embed_dim // 2, embed_dim
+        ]
+        self.conv_cfg = conv_cfg
+        self.norm_cfg = norm_cfg
+
+        self.patch_embed = nn.Sequential()
+
+        for i in range(len(self.input_channels)):
+            conv_bn = Conv_BN(
+                self.input_channels[i],
+                self.output_channels[i],
+                kernel_size=kernel_size,
+                stride=stride,
+                pad=pad,
+                dilation=dilation,
+                groups=groups,
+                bn_weight_init=bn_weight_init,
+                conv_cfg=conv_cfg,
+                norm_cfg=norm_cfg,
+            )
+            self.patch_embed.add_module('%d' % (2 * i), conv_bn)
+            if i < len(self.input_channels) - 1:
+                self.patch_embed.add_module('%d' % (i * 2 + 1), activation())
+
+    def forward(self, x):
+        x = self.patch_embed(x)
+        return x
+
+
+class Conv_BN(nn.Sequential):
```

Comment:
The name of classes should obey [Google python style](https://zh-google-styleguide.readthedocs.io/en/latest/google-python-styleguide/python_style_rules/#section-15)

Reply:
Modified